<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>6820</width>
    <height>12120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0,0,0);</string>
  </property>
  <widget class="QPushButton" name="showover_btn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>1570</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(75, 75, 75);
font: 20pt &quot;HY견고딕&quot;;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>영상 ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setplan_btn">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>1570</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(75, 75, 75);
font: 20pt &quot;HY견고딕&quot;;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>운동 옵션</string>
   </property>
  </widget>
  <widget class="QLabel" name="kg_lb">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY견고딕</family>
     <pointsize>50</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 50pt &quot;HY견고딕&quot;;

background-color: rgb(32, 32, 32);</string>
   </property>
   <property name="text">
    <string>KG</string>
   </property>
  </widget>
  <widget class="QPushButton" name="home_btn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>1700</y>
     <width>121</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(img/home_btn.png);
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>1700</y>
     <width>111</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(img/back_btn.png);
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="tip_btn">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>1700</y>
     <width>111</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(img/info_btn.png);
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="datetime_lb">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY견고딕</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>날짜, 시간</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY견고딕</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Virtual Fitness</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>510</y>
     <width>481</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0,0,0);
font: 20pt &quot;HY견고딕&quot;;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>발판을 밀면 운동이 시작됩니다.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTopCameraView">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>240</y>
     <width>879</width>
     <height>451</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>600</y>
     <width>891</width>
     <height>761</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="viewVertLayout">
    <item>
     <widget class="QTableWidget" name="weightTable">
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <column/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>1091</width>
     <height>1611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(32, 32, 32);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="planer_6_btn">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>110</y>
     <width>201</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(img/plan_6,7.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="countBox_container" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1610</y>
     <width>1081</width>
     <height>191</height>
    </rect>
   </property>
   <widget class="QPushButton" name="mute_btn_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(img/muteon_5.png);
background-color: rgb(0,0,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rep_count_lb">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 40pt &quot;HY견고딕&quot;;
color: rgb(124, 251, 217);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;HY견고딕&quot;;</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLabel" name="rep_set_lb">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;HY견고딕&quot;;</string>
    </property>
    <property name="text">
     <string>12</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>60</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
image: url(img/set_excellent_6,7.png);
background-color: rgb(0,0,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="countBox_container_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1370</y>
     <width>1081</width>
     <height>191</height>
    </rect>
   </property>
   <widget class="QPushButton" name="mute_btn_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(img/muteon_5.png);
background-color: rgb(0,0,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>511</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0,0,0);
font: 20pt &quot;HY견고딕&quot;;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>다이얼을 돌려 원하는 무게를 설정하세요</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDial" name="weight_dial">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>20</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="notchTarget">
     <double>10.000000000000000</double>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <zorder>label_4</zorder>
  <zorder>showover_btn</zorder>
  <zorder>setplan_btn</zorder>
  <zorder>kg_lb</zorder>
  <zorder>home_btn</zorder>
  <zorder>back_btn</zorder>
  <zorder>tip_btn</zorder>
  <zorder>datetime_lb</zorder>
  <zorder>label_5</zorder>
  <zorder>label_2</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>lblTopCameraView</zorder>
  <zorder>planer_6_btn</zorder>
  <zorder>countBox_container</zorder>
  <zorder>countBox_container_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
